tests/js/client/server_parameters/test-web-interface-on.js

```diff
@@ -0,0 +1,65 @@
+/*jshint globalstrict:false, strict:false */
```

Comment:
if supported, please join the foxx on/Webif Off testcase (and vice versa) to one test each to reduce test resources.